Macros/e3 Includes/e3_Basics.inc

```diff
@@ -2054,6 +2054,9 @@ SUB check_Supply
 	/if (${Debug}) /echo |- check_Supply ==>
   /if (${Waiting4Rez}) /return
   /if (${Bool[${Me.Buff[Resurrection Sickness]}]}) /return
+  | Extra checks here because sometimes Waiting4Rez doesn't seem to short out supply
+  /if (!${Bool[${InvSlot[pack8].Item.ID}]}) /return 
+  /if (${Me.MaxHPs} < 10000) /return
```

Comment:
I would check to see if armor slots are there or not. 10khp would be most people as they are leveling

Reply:
and a combination of no buffs probably as well

Reply:
I figured checking 2 equipped items and then checking for either 0 buffs or 1 buff which is xp pot should be good.